acquia_lift_profiles/acquia_lift_profiles.admin.inc

```diff
@@ -6,6 +6,86 @@
  */
 
 /**
+ * Admin form for configuring lift_event_values behavior.
+ */
+function acquia_lift_profiles_lift_event_values_form($form, &$form_state) {
+  $form = array(
+    '#attached' => array(
+      'css' => array(
+        drupal_get_path('module', 'acquia_lift') . '/css/acquia_lift.admin.css',
+      ),
+    ),
+  );
+
+  $form['lift_event_values'] = array(
+    '#tree' => TRUE,
+    '#theme' => 'acquia_lift_profiles_lift_event_value_form_table',
+  );
+  $serialized_lift_event_values = variable_get('acquia_lift_profiles_lift_event_values', 'a:0:{}');
```

Comment:
There were 3 possible ways to store it and I ended trying all three. Storing in variable seemed to be the best one because:

* Storing in visitor_actions_actions table's data field blob is not okay because we want to have lift event values even for the stock / system actions such as user_login and form_submit, and for those do not have a record in the table to piggy-back on. Also it requires "hacking" the visitor_actions code to retain the data in the data-field and prevent it being overwritten when the action is been updated somewhere else.
* Storing in a new table acquia_lift_profiles_actions is an inferior approach because:
   * More code, a more table.
   * For performance reasons, we still need to pulling up the whole table every time we need to use it (see acquia_lift_profiles_visitor_actions_page_build()).